src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -584,6 +584,12 @@ public bool Paste()
                 {
                     InsertCode(format);
                 }
+                else if (Clipboard.ContainsData(ReplFormat))
+                {
+                    var blocks = BufferBlock.Deserialize((string)Clipboard.GetData(ReplFormat));
+                    var text = GetTextFromTextBlocks(blocks);
```

Comment:
Shouldn't each input be pasted separately?  What if one is a REPL command (e.g. ```#help``` or ```#clear```)?